gulp/tasks/flow.js

```diff
@@ -1,11 +1,30 @@
 var gulp = require('gulp');
 var execFile = require('child_process').execFile;
 
-gulp.task('flow-win', function() {
-  execFile('./flow/flow.exe', ['check','--lib','./flow/libs','--strip-root','./flow'],function (err, stdout, stderr) {
-    if(err) console.log('Error:' + err);
-    if(stdout) console.log(stdout);
-  });
+gulp.task('flow', function() {
+  if (process && process.platform === 'win32') {
+    //due to flow-bin not working on windows (because there isnt an official build https://github.com/facebook/flow/issues/6)
+    execFile('./flow/flow.exe', ['check','--lib','./flow/libs','--strip-root','./flow'],function (err, stdout, stderr) {
+      if(err) console.log('Error:' + err);
+      if(stdout) console.log(stdout);
+    });
+  } else {
+    //this is how we WANT to run flow
+    var flow = require('gulp-flowtype');
```

Comment:
does this actually have flow installed, or does it require flow to be setup on local machine before?